_inc/client/components/form/form-toggle/index.jsx

```diff
@@ -112,39 +117,27 @@ export default class FormToggle extends Component {
 	};
 
 	render() {
-		const id = this.props.id || 'toggle-' + this.id;
+		const DisabledComponent = this.props.disabled ? Disabled : Fragment;
+		const labelContent = (
+			<span className="form-toggle__label-content" onClick={ this.onLabelClick }>
+				{ this.props.children }
+			</span>
+		);
 		const toggleClasses = classNames( 'form-toggle', this.props.className, {
 			'is-toggling': this.props.toggling,
+			id: this.props.id || 'toggle-' + this.id,
```

Comment:
This doesn't seem to belong here - it will add an `id` className. Did we mean to delete it, since `ToggleControl` will generate its own ID?